package.json

```diff
@@ -32,6 +32,7 @@
     "semver": "^5.3.0",
     "sosa_mongo": "^1.0.3",
     "timebucket": "^0.4.0",
+    "talib": "^1.0.3",
```

Comment:
Although, yes, that sucks, TA-Lib has many indicators and many financial functions that might be of use.  This could be a case where I'd say that we drop Windows support.